clusterloader2/pkg/measurement/common/wait_for_pods.go

```diff
@@ -91,10 +91,12 @@ func waitForPods(clientSet clientset.Interface, namespace, labelSelector, fieldS
 				runningPodsCount++
 			}
 		}
-		if namespace == metav1.NamespaceAll {
-			glog.Infof("WaitForRunningPods: running %d / %d", runningPodsCount, desiredPodCount)
-		} else {
-			glog.Infof("WaitForRunningPods: %s: running %d / %d", namespace, runningPodsCount, desiredPodCount)
+		if log {
+			if namespace == metav1.NamespaceAll {
+				glog.Infof("WaitForRunningPods: running %d / %d", runningPodsCount, desiredPodCount)
```

Comment:
IMO you should also include label and field selectors to the log. Otherwise they won't be much informative I think, when there are many calls to this function.

Reply:
I agree with you that we should be able to identify measurement.
IMHO I'm not sure that putting every parameter that function have to the log is the best option. Maybe we want to have logs like "<metric_type>(<metric_identifier>): <log text>"? We already provide identifiers in test definitions. WYT?

Reply:
Label selector and field selector added to the log.